{"name": "test_auditTrailUnitAdd", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'count' referenced before assignment", "trace": "def test_auditTrailUnitAdd():\n>       TestCoverageUnit.auditTrailUnitAdd(driver,oldName,oldDescription,\"No\")\n\nTestCaseUnit.py:54: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTestCoverage\\TestCoverageUnit.py:369: in auditTrailUnitAdd\n    beforeCount=TestCoverageAudittrail.auditTrailRecordCount(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f63f478b1239c4c359646510d252dfff\")>\n\n    def auditTrailRecordCount(driver):\n    \n        ScreenNavigate.auditTrailPreCondition(driver)\n    \n        try:\n            countText= driver.find_element(By.XPATH,\"(//*[@class='k-pager-info k-label'])[2]\").text\n    \n            individualText = countText.split(' ')\n    \n            count = individualText[4]\n    \n            logger.info(\"The count is fetched, it shows {} record in the grid\".format(count))\n    \n            count = int(count)\n    \n        except:\n    \n            try:\n                time.sleep(3)\n    \n                countText = driver.find_element(By.XPATH, \"(//*[@class='k-pager-info k-label'])[2]\").text\n    \n                individualText = countText.split(' ')\n    \n                count = individualText[4]\n    \n                logger.info(\"The count is fetched, it shows {} record in the grid\".format(count))\n    \n                count = int(count)\n    \n            except:\n    \n                logger.error(\"Unable to get the count in the Audit trail screen\")\n    \n    \n        ScreenNavigate.auditTrailPostCondition(driver)\n    \n>       return count\nE       UnboundLocalError: local variable 'count' referenced before assignment\n\nTestCoverage\\TestCoverageAudittrail.py:52: UnboundLocalError"}, "attachments": [{"name": "log", "source": "82ce0a47-62cb-41ee-9d50-130b3abec886-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "09e57c02-482f-4cb9-bf50-1e33d7083544-attachment.txt", "type": "text/plain"}], "start": 1650621087384, "stop": 1650621165509, "uuid": "c7326c57-d8db-4aba-b439-85ae2d1be511", "historyId": "8e44131e7c2b648eb351c090d10dcad7", "testCaseId": "f5a9c943acd3daab2b8e9ddb3034b3e7", "fullName": "TestCaseUnit#test_auditTrailUnitAdd", "labels": [{"name": "suite", "value": "TestCaseUnit"}, {"name": "host", "value": "AGL78"}, {"name": "thread", "value": "21384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCaseUnit"}]}